BUILDING.md

```diff
@@ -0,0 +1,79 @@
+# Building JanusGraph client for Python
+
+-   Building Library:
+
+    A set of scripts have been created to make the job of building the library easier for user, but the scripts
+    expects few pre requisites to be installed before hand. 
+
+-   Pre-Requisites:
+
+    -   Python needs to be installed on system. The library is currently tested against Python
+        3.4, 3.5 & 3.6 only.
+
+    -   Python is added to system path. Make sure, the user can run the following commands from command line:
+        -   python3/ python3.4/ python3.5/ python3.6 on Unix machines.
+        -   python on Windows machines.
+
+    -   Virtualenv needs to be installed. See [PyPi Page](https://pypi.org/project/virtualenv/) for installation steps.
+
+    -   The automated build scripts are tested on following Operating Systems:
+        -   Windows 7
+        -   CentOS
+        -   RHEL 7
+        -   Ubuntu 18.04
+        -   MacOS
+
+-   Building on Windows 7:
+
+    -   A script build.cmd is created which accepts following keyword arguments:
+        -   `d`: Specifies whether documents to be build. Set to true or false.
+        -   `b`: Specifies whether library to be build. Set to true or false.
+
+    -   If any of parameters are missing, the script exits with error message. 
+        An appropriate way to invoke the script would be:
+        ```bash
+        build.cmd "d=true" "b=true"
+        ```
+
+    -   The above example builds the docs, library and also installs to local global Python site-packages.
+
+-   Building on Unix System:
+
+    -   A script named build.sh is created which accepts following parameters:
```

Comment:
I just tried running just `./build.sh` (without any parameters) on my Ubuntu in the WSL, but I'm getting permission denied errors in the steps `Running sphinx_html` and also in `Building distribution in [DIR]`. Do I need to do anything else? Or is this only not working on my machine / the WSL?

Reply:
@FlorianHockmann the Pybuilder was updated since I last ran the build, and hence you might be getting errors. I tried on CentOS VM myself which was working last time and it started failing so did it Ubuntu WSL.

I just verified it now works on CentOS and Ubuntu WSL. It works :-)
Though the strange thing is I didnt get any Permission Denied errors. I was getting some diff kinda error. Can you pull the current version and verify?

Reply:
I just tried it again and now the docs build successfully but I still get the error when building the lib. However, this could very well be an issue specific to the WSL on my machine or something like that as I could successfully build this with the `python:3.6-buster` image. So it looks like everything is working correctly in general and I would treat this as a problem with my machine right now.